app/.env.docker-example

```diff
@@ -7,12 +7,12 @@ DJANGO_SUPERUSER_EMAIL=admin@admin.com
 DJANGO_SUPERUSER_PASSWORD=admin
 
 # postgres settings for docker
-SQL_ENGINE=django.db.backends.postgresql
-SQL_DATABASE=people_depot_dev
-SQL_USER=people_depot
-SQL_PASSWORD=people_depot
-SQL_HOST=db
-SQL_PORT=5432
+POSTGRES_ENGINE=django.db.backends.postgresql
+POSTGRES_DATABASE=people_depot_dev
```

Comment:
I think the variable name is actually POSTGRES_DB, from the `docker-compose.yml`